pkg/file/serverless.go

```diff
@@ -49,18 +50,38 @@ type Schedule struct {
 	Data string
 }
 
+var Aos = afero.NewOsFs()
+
 // ParseServerless accepts serverless yaml file path and returns decoded structure
 func ParseManifest(path string) (Definition, error) {
-	var f Definition
-	if _, err := os.Stat(path); err != nil {
-		return f, err
+	var definition Definition
+
+	exists, err := afero.Exists(Aos, path)
+
+	if !exists || err != nil {
+		return definition, errors.New("could not find manifest file")
 	}
-	data, err := ioutil.ReadFile(path)
+
+	data, err := afero.ReadFile(Aos, path)
+
 	if err != nil {
-		return f, err
+		return definition, err
+	}
+
+	definition.Repository = filepath.Base(filepath.Dir(path))
+	err = yaml.Unmarshal(data, &definition)
```

Comment:
I wonder if we should use `UnmarshalStrict()`. Silently ignoring erroneous configuration in `serverless.yml` might cause some unexpected consequences :thinking:

Reply:
makes sense, let's use `UnmarshalStrict()`